### DISCUSSION TITLE
Asp.Net Core ILogger and Serilog.Sinks.Email

### DISCUSSION THREAD

**Discussion by chrisklepeis** [RehanSaeed/Serilog.Exceptions#6] [Q&A]
Not sure how to get this working with the built in ILogger in ASP.Net Core or if it can even integrate with the Serilog.Sinks.Email package.

```
        Log.Logger = new LoggerConfiguration().Enrich.WithExceptionDetails()
            .WriteTo.Email(new EmailConnectionInfo()
            {
                // my smtp info here
            }, restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Error)
            .WriteTo.Trace().CreateLogger();
```

Then in my controller I try:
`_logger.LogError("Error occured", ex); // where _logger is the ILogger from ASP.Net Core, not Serilog..`

I do get the email, but nothing related to an exception. 

Maybe we need an Asp.Net Core extension method for ILogger to call Error, passing the exception. Just a thought. I'm using Serilog for ASP.Net Core from the readme here https://github.com/serilog/serilog-extensions-logging 

Edit... found out I was using the wrong override of "LogError"
`_logger.LogError(new EventId(1, "Error"), ex, "Error");`

works.

**Comment by RehanSaeed** [comment]
Glad you got it working sir!
